app/models/course/group_user.rb

```diff
@@ -16,6 +16,12 @@ class Course::GroupUser < ApplicationRecord
   belongs_to :course_user, inverse_of: :group_users
   belongs_to :group, class_name: Course::Group.name, inverse_of: :group_users
 
+  scope :order_by_course_user_name, lambda {
+                                      joins('LEFT OUTER JOIN course_users ON '\
+                                            'course_users.id = course_group_users.course_user_id').
+                                        order('name DESC')
```

Comment:
Hmm is this the same as `course_users.name ASC`?

Reply:
oops should be ASC.. updated thanks